app/src/main/java/com/example/africanid/MainActivity.java

```diff
@@ -3,12 +3,64 @@
 import androidx.appcompat.app.AppCompatActivity;
 
 import android.os.Bundle;
+import android.view.View;
+import android.widget.Button;
+import android.widget.EditText;
+import android.widget.TextView;
+import android.widget.Toast;
 
 public class MainActivity extends AppCompatActivity {
 
     @Override
     protected void onCreate(Bundle savedInstanceState) {
         super.onCreate(savedInstanceState);
         setContentView(R.layout.activity_main);
+
+        Button button = findViewById(R.id.buttonS);
+
+        final TextView label1 = findViewById(R.id.textLabel1);
+        final TextView label2 = findViewById(R.id.textLabel2);
+        final TextView txtBirthday = (TextView)findViewById(R.id.textBirthday);
+        final TextView txtGender = (TextView)findViewById(R.id.textGender);
+
+        label1.setVisibility(View.INVISIBLE);
+        label2.setVisibility(View.INVISIBLE);
+        txtBirthday.setVisibility(View.INVISIBLE);
+        txtGender.setVisibility(View.INVISIBLE);
+
+
```

Comment:
Here you have a blank space without any objective, could please standardize your number of spaces?